dataset/.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "python.pythonPath": "/u/pjuangph/.conda/envs/dev/bin/python"
+    "python.pythonPath": "C:\\Users\\Paht\\anaconda3\\envs\\dev\\python.exe"
```

Comment:
Its already being tracked and this repo is getting merged into data collection at some point, so it doesn't matter, but the .vscode folder should probably be in the .gitignore file so it doesn't overwrite other's configs.